tests/BtmsGateway.IntegrationTests/Utils/Http/ProxyTests.cs

```diff
@@ -33,24 +33,24 @@ public async Task When_event_processed_and_post_to_cds_takes_longer_than_http_cl
     {
         await wireMockClient.ResetWiremock();
 
-        var putMappingBuilder = _wireMockAdminApi.GetMappingBuilder();
-        putMappingBuilder.Given(m =>
-            m.WithRequest(req => req.UsingPut().WithPath($"/comparer/btms-decisions/{_mrn}"))
-                .WithResponse(rsp => rsp.WithStatusCode(HttpStatusCode.OK).WithBody(_decisionNotification))
-        );
-        var putStatus = await putMappingBuilder.BuildAndPostAsync();
-        Assert.NotNull(putStatus.Guid);
+        ////var putMappingBuilder = _wireMockAdminApi.GetMappingBuilder();
+        ////putMappingBuilder.Given(m =>
+        ////    m.WithRequest(req => req.UsingPut().WithPath($"/comparer/btms-decisions/{_mrn}"))
+        ////        .WithResponse(rsp => rsp.WithStatusCode(HttpStatusCode.OK).WithBody(_decisionNotification))
+        ////);
+        ////var putStatus = await putMappingBuilder.BuildAndPostAsync();
+        ////Assert.NotNull(putStatus.Guid);
 
-        var postMappingBuilder = _wireMockAdminApi.GetMappingBuilder();
-        postMappingBuilder.Given(m =>
-            m.WithRequest(req => req.UsingPost().WithPath("/cds/ws/CDS/defra/alvsclearanceinbound/v1"))
-                .WithResponse(rsp =>
-                    rsp.WithStatusCode(HttpStatusCode.ServiceUnavailable)
-                        .WithDelay(TimeSpan.FromSeconds(Proxy.DefaultHttpClientTimeoutSeconds + 1))
-                )
-        );
-        var postStatus = await postMappingBuilder.BuildAndPostAsync();
-        Assert.NotNull(postStatus.Guid);
+        ////var postMappingBuilder = _wireMockAdminApi.GetMappingBuilder();
```

Comment:
The alvsclearanceinbound/v1 is the endpoint that is hit when communicating decisions/errors to EIS.
I think this block may still be required?